csharp-intermediate-topics/FindTheLatitudeAndLongitudeOfALocationInCSharp/FindTheLatitudeAndLongitudeOfALocationInCSharp/Services/GoogleLocationService.cs

```diff
@@ -0,0 +1,20 @@
+using GoogleMaps.LocationServices;
+using Models;
```

Comment:
@elliezub 
We should also remove unused `usings` here

Reply:
These two seem like they need to be here but I was able to remove using Services;